desc/objectives/_fast_ion.py

```diff
@@ -91,22 +95,22 @@ class GammaC(_Objective):
         irrational magnetic surface better, with diminishing returns.
     num_well : int
         Maximum number of wells to detect for each pitch and field line.
-        Giving ``None`` will detect all wells but due to current limitations in
+        Giving ``-1`` will detect all wells but due to current limitations in
```

Comment:
is the default of `None` the same as `-1` or does it use the `A+iota*C` heuristic?

Reply:
In this PR, I have changed the default `None` to be very safe heuristic that we set. If the user has unlimited compute, and does not trust our heurstic, they can set it to `-1` to ensure all wells are found.